artpaint/paintwindow/ImageView.cpp

```diff
@@ -1419,7 +1419,7 @@ ImageView::PaintToolThread()
 
 	int32 tool_type = ToolManager::Instance().ReturnActiveToolType();
 
-	if (modifiers() & B_COMMAND_KEY)
+	if (modifiers() & B_CONTROL_KEY) {
```

Comment:
This stray curly foils the build. :)

Reply:
dang. that was a merge conflict...I swear I don't usually push changes that don't build (aside from the 32-bit casts)

Reply:
fixed, and my face is red.